bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeDataManager.java

```diff
@@ -184,8 +182,25 @@ protected List<String> getCommandArgsArchives(String pythonExecutablenPath, IFil
 		arguments.add(IDFUtil.getIDFSizeScriptFile().getAbsolutePath());
 		arguments.add(file.getLocation().toOSString());
 		arguments.add("--archives"); //$NON-NLS-1$
-		arguments.add("--json"); //$NON-NLS-1$
+		arguments.addAll(addJsonParseCommand());
+
+		return arguments;
+	}
 
+	private List<String> addJsonParseCommand()
+	{
+		List<String> arguments = new ArrayList<String>();
+		IEnvironmentVariable idfVersionEnv = new IDFEnvironmentVariables().getEnv("ESP_IDF_VERSION"); //$NON-NLS-1$
```

Comment:
Hi @sigmaaa Are we updating `ESP_IDF_VERSION` with esp-idf installation and configuration?

Reply:
Hi @kolipakakondal. No, we are not updating it. I'll fix that. Thanks

Reply:
fixed